proxies/live/apiproxy/policies/OAuthV2.TokenExchangeGenerateAccessTokenFilteredScopes.xml

```diff
@@ -23,7 +23,7 @@
         <Attribute name="id_token-issuer" ref="jwt.VerifyJWT.SubjectToken.claim.issuer" display="true" />
         <Attribute name="id_token-acr" ref="jwt.DecodeJWT.FromExternalIdToken.claim.acr" display="true" />
         <Attribute name="id_token-amr" ref="jwt.DecodeJWT.FromExternalIdToken.claim.amr" display="true" />
-        <Attribute name="id_assurance_level" ref="jwt.DecodeJWT.FromExternalIdToken.claim.id_assurance_level" display="true" />
+        <Attribute name="id_token-id-assurance-level" ref="jwt.DecodeJWT.FromExternalIdToken.claim.id_assurance_level" display="true" />
```

Comment:
Just thought, do you have to get acr/amr/id_assurance_level from the JWT in the subject token or does this work ok? Just thinking that id_token-issuer uses "jwt.VerifyJWT.SubjectToken.claim.issuer"